docassemble/ALToolbox/al_income.py

```diff
@@ -958,12 +959,18 @@ class ALItemizedJob(DAObject):
         represents how frequently the income is earned
     .is_hourly {bool} (Optional) Whether the value represents a figure that the
         user earns on an hourly basis, rather than for the full time period
-    .hours_per_period {int} (Optional) If the job is hourly, how many hours the
-        user works per period.
+    .hours_per_period {float | Decimal} (Optional) If the job is hourly, how
+        many hours the user works per period.
+    .is_seasonal {bool} (Optional) Whether the job's income changes drastically
+        during different times of year.
     .employer {Individual} (Optional) Individual assumed to have a name and,
         optionally, an address and phone.
     .source {str} (Optional) The category of this item, like "public service".
-        Defaults to "job".
+    .months {ALItemizedJobMonthList} Automatically exist, but they won't be used
```

Comment:
Question: is this meant to be `months`  or `paystubs`?

Reply:
I think it depends on where we go with this and how granular we let people get (as you discuss elsewhere) and how much we want to let them customize.